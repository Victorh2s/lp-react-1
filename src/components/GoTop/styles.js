import styled, { css } from 'styled-components';

export const Container = styled.a`
  ${({ theme }) => css`
    position: fixed;
    background: none;
    color: ${theme.colors.colorbuttons};
    border: 0.1rem solid ${theme.colors.colorbuttons};
    border-radius: 5rem;
    display: flex;
    align-items: center;
    justify-content: center;
    width: 4rem;
    height: 4rem;
    bottom: 2rem;
    right: 2rem;
    z-index: 6;
  `}
`;
