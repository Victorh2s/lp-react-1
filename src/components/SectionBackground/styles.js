import styled, { css } from 'styled-components';

const backgroundActive = (theme) => css`
  background: ${theme.colors.primaryColor};
  color: ${theme.colors.white};
`;

export const Container = styled.div`
${({ theme, background }) => css`
transition: all 300ms ease-in-out;
background: ${theme.colors.white};
color: ${theme.colors.primaryColor};
${background && backgroundActive(theme)}
min-height: 80vh;
display: flex;
align-items: center;
`}
`;
