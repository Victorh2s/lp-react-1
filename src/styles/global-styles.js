import { createGlobalStyle, css } from 'styled-components';

export const GlobalStyles = createGlobalStyle`
  @import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,400;0,600;0,800;1,700&display=swap');
  * {
    margin:0;
    padding:0;
  }
  html {
    font-size: 62.5%;
  }
  body {
    font-size:2.4rem;
    font-family: ${({theme})=> theme.font,family.default};
  }
  h1 {
    font-size: 2.4rem;
    font-family:  ${({theme})=> theme.font,family.secondary};
  }
`;
