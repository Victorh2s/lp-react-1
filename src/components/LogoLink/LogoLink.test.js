import { screen } from '@testing-library/react';
import { LogoLink } from '.';
import { renderTheme } from '../../styles/render-theme';
describe('<LogoLink/>', () => {
  it('should render', () => {
    renderTheme(<LogoLink>Children</LogoLink>);
    const logolink = screen.getByRole('heading');
    expect(logolink).toBeInTheDocument();
  });
});
