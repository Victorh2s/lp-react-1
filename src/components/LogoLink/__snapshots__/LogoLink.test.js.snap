// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LogoLink /> should match snapshot 1`] = `
<h1
  class="sc-gsnTZi gGAKok"
>
  <a
    class="sc-bczRLJ dqLDEa"
    href="#target"
  >
    <img
      alt="OlÃ¡ mundo"
      src="image.jpg"
    />
  </a>
</h1>
`;
